Makefile

```diff
@@ -0,0 +1,38 @@
+.DEFAULT_GOAL = help
+
+.PHONY: sh
+sh: ## Get a terminal with Bash.
+	docker-compose exec server bash
+
+.PHONY: logs
+logs: ## Tail the logs, locally.
+	docker-compose logs -f
+
+.PHONY: up
+up: ## Spin up a dev suite of containers.
+	docker-compose up -d
+
+.PHONY: generate
+generate: ## a one time use command needed to setup the db after the creation of the containers
+	docker-compose exec -ti server npx prisma generate --schema server/prisma/schema.prisma 
+
+.PHONY: migrate
+migrate: ## push db changes to the db.
+	docker-compose exec -ti server npx prisma db push --schema server/prisma/schema.prisma
+
+.PHONY: down
+down: ## Stops up containers.
+	docker-compose down
+
+.PHONY: db
+db: ## attach to the db cli
+	docker-compose exec -ti mongo mongo --host=mongo --port=27018 -u "mongoadmin" -p "secret"
```

Comment:
this connects you to a mongo cli where you can query your db. it's actually pretty neat and the [docs](https://www.mongodb.com/docs/v4.4/reference/mongo-shell/) are pretty nice.